mongodb-setup/app/server/db.server.ts

```diff
@@ -0,0 +1,38 @@
+import * as dotenv from 'dotenv';
```

Comment:
There's a shorter import for doing this

```suggestion
import from 'dotenv/config';
```

Reply:
Hello @MichaelDeBoey , I receive this error

![error importing](https://i.postimg.cc/76GcnP0D/Screenshot-2023-07-17-at-20-29-07.png)